.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Scala",
+  "dockerFile": "Dockerfile",
+  "settings": {
+    "terminal.integrated.shell.linux": "/bin/bash",
+    "[scala]": {
+      "editor.tabSize": 2,
+      "editor.insertSpaces": true
+    }
+  },
+  "extensions": [
+    "scala-lang.scala", "zxh404.vscode-proto3"
```

Comment:
I did not append more extensions here. One might install her preferred extensions (metals?) for Scala and the like through [`.dotfiles`](https://docs.microsoft.com/en-us/visualstudio/codespaces/reference/personalizing).